tests/tenant-fetcher/tests/handler_test.go

```diff
@@ -236,6 +236,40 @@ func TestRegionalOnboardingHandler(t *testing.T) {
 			require.NoError(t, err)
 			assertTenant(t, tenant, childTenant.SubaccountID, childTenant.Subdomain)
 			require.Equal(t, regionPathParamValue, tenant.Labels["region"])
+
+			parentTenantAfterInsert, err := fixtures.GetTenantByExternalID(dexGraphQLClient, parentTenant.TenantID)
+			require.NoError(t, err)
+			assertTenant(t, parentTenantAfterInsert, parentTenant.TenantID, parentTenant.Subdomain)
+			require.Equal(t, regionPathParamValue, parentTenantAfterInsert.Labels["region"])
```

Comment:
Add this check to the checks directly after inserting the parent (before inserting the child) as well.